### PR TITLE
[pulsar-perf] Write histogram files for consume command

### PR DESCRIPTION
**Pull request by mfleming**
### Motivation

The `consume` command for `pulsar-perf` doesn't generate histogram file (`.hgrm` files) like its counterpart `produce` which makes it more difficult to automate collection of consumer performance results and basically necessitates sending metrics to prometheus, etc.

### Modifications

Write the contents of the histogram to a `.hgrm` file like the `produce` command.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.
- [x] This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

No

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [x] `no-need-doc` 
  
The change brings behaviour inline with the `produce` command and I'm not sure additional docs are needed.

### CODE REVIEW

File: pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java [hunk 1]
```diff
@@ -407,6 +410,16 @@ public static void main(String[] args) throws Exception {
 
         Histogram reportHistogram = null;
 
+        String statsFileName = "perf-consumer-" + System.currentTimeMillis() + ".hgrm";
```
Comment (merlimat) [inline pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java:413]: This should probably be turned off by default

Comment (mfleming) [reply pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java:413]: Should it be disabled by default to maintain backwards compatible behaviour before this change? My intention was to make both `produce` and `consume` behave the same way and this functionality is default enabled for `produce`.

Comment (eolivelli) [reply pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java:413]: I agree

**Review by eolivelli — CHANGES_REQUESTED** [review]
Thank you very much for your contribution.

I second @merlimat 's comments.
Please address those comments and we are good to ho as soon as CI passes

**Review by merlimat — APPROVED** [review]
👍

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@eolivelli Please help review the PR.

**Review by eolivelli — APPROVED** [review]
Lgtm

**Pull request merged**